src/Renderer/LayeredMaterial.js

```diff
@@ -139,218 +82,210 @@ const LayeredMaterial = function LayeredMaterial(options) {
     this.uniforms.colorLayersCount = new THREE.Uniform(1);
 
     // Layer setting
-    // Offset color texture slot | Projection | fx | Opacity
+    // visibility | opacity | fx
     this.uniforms.paramLayers = new THREE.Uniform(paramLayers);
 
     // Elevation texture cropping
     this.uniforms.offsetScale_L00 = new THREE.Uniform(this.offsetScale[l_ELEVATION]);
 
-    // Color texture cropping
-    this.uniforms.offsetScale_L01 = new THREE.Uniform(this.offsetScale[l_COLOR]);
-
     // Light position
     this.uniforms.lightPosition = new THREE.Uniform(new THREE.Vector3(-0.5, 0.0, 1.0));
 
     this.colorLayersId = [];
     this.elevationLayersId = [];
 
-    if (__DEBUG__) {
-        this.checkLayersConsistency = function checkLayersConsistency(node, imageryLayers) {
-            for (const layer of imageryLayers) {
-                const index = this.indexOfColorLayer(layer.id);
-                if (index < 0) {
-                    continue;
-                }
-
-                const offset = this.getTextureOffsetByLayerIndex(index);
-                const count = this.getTextureCountByLayerIndex(index);
-                let total = 0;
-                for (let i = 0; i < this.loadedTexturesCount[1]; i++) {
-                    if (!this.uniforms.dTextures_01.value[i].image) {
-                        throw new Error(`${node.id} - Missing texture at index ${i} for layer ${layer.id}`);
-                    }
-
-                    const critere1 = (offset <= i && i < (offset + count));
-                    const search = layer.name ? `LAYERS=${layer.name}&` : `LAYER=${layer.options.name}&`;
-                    const critere2 = this.uniforms.dTextures_01.value[i].image.currentSrc.indexOf(search) > 0;
-
-                    if (critere1 && !critere2) {
-                        throw new Error(`${node.id} - Texture should belong to ${layer.id} but comes from ${this.uniforms.dTextures_01.value[i].image.currentSrc}`);
-                    } else if (!critere1 && critere2) {
-                        throw new Error(`${node.id} - Texture shouldn't belong to ${layer.id}`);
-                    } else if (critere1) {
-                        total++;
-                    }
-                }
-                if (total != count) {
-                    throw new Error(`${node.id} - Invalid total texture count. Found: ${total}, expected: ${count} for ${layer.id}`);
-                }
-            }
-        };
-    }
+    const atlasTextures = Array(8);
```

Comment:
it's http://eslint.org/docs/rules/new-cap, currently disabled in our conf (but apparently they have exceptions for `Array`, so let's forget about this one)